sdk/monitor/Azure.Monitor.OpenTelemetry/src/AzureMonitorOpenTelemetryImplementations.cs

```diff
@@ -37,6 +38,26 @@ internal static IServiceCollection AddAzureMonitorOpenTelemetryWithAction(IServi
                             .AddHttpClientInstrumentation()
                             .AddAzureMonitorMetricExporter());
 
+            services.AddLogging(logging =>
+            {
+                AzureMonitorOpenTelemetryOptions logExporterOptions = new();
+                if (configureAzureMonitorOpenTelemetry != null)
+                {
+                    configureAzureMonitorOpenTelemetry(logExporterOptions);
+                }
+
+                if (logExporterOptions.EnableLogs)
+                {
+                    logging.AddOpenTelemetry(builderOptions =>
+                    {
+                        builderOptions.IncludeFormattedMessage = true;
+                        builderOptions.ParseStateValues = true;
+                        builderOptions.IncludeScopes = true;
```

Comment:
```suggestion
                        builderOptions.IncludeScopes = false;
```

Scopes add a lot of extra information, a lot of them from asp.net core itself which are duplicate info.. I'd vote to keep this opt-in behavior.